plugins/lightspeed/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@janus-idp/backstage-plugin-lightspeed",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
```

Comment:
@rohitkrai03 here we need to add `pluginId` and `supported-versions` similar to all other plugins to avoid running into this issue https://issues.redhat.com//browse/RHIDP-3236.

Reply:
Updated.